src/scripts/color_palette.json

```diff
@@ -1,6 +1,6 @@
 {
     "Chemist": "#F7DC6F",
-    "o1": "#AB8D9E",
+    "GPT-4o1": "#AB8D9E",
```

Comment:
I think the official name really is o1?

Reply:
I think they tried to break with the GPT naming to indicate a big difference

Reply:
Yes, or OpenAI O1. I do not know when and how I changed this

Reply:
So, yeah I agree that this should be reverted